msgs/confirmSigMsg.go

```diff
@@ -0,0 +1,74 @@
+package msgs
+
+import (
+	"github.com/FourthState/plasma-mvp-sidechain/plasma"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/FourthState/plasma-mvp-sidechain/utils"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+const (
+	// ConfirmSigMsgRoute is used for routing this message.
+	ConfirmSigMsgRoute = "confirmSig"
+)
+
+// SpendMsg implements the RLP interface through `Transaction`
+type ConfirmSigMsg struct {
```

Comment:
I like the idea of re-using `plasma.Input`. Allows you to pass confirmsignatures for unrelated inputs.

One blocker here though is that `plasma.Input` requires the transaction signature to also be present. You _can_ argue that the transaction signatures should not be apart of `plasma.Input` and instead `plasma.Transaction`. `input.ValidateBasic` will fail if the signatures are empty.

Reply:
Applied input.signatures = byte(1) for now